### ISSUE TITLE
Array type handling: Normalization

### ISSUE THREAD

**Issue by eroell** [theislab/ehrapy#834]
### Description of feature

Ensure the normalization methods handle the array types (numpy ndarray, dask array, check if we want more?) consistently, and properly fail if an unsupported array is encountered.

`log_norm`
- [ ] Consistent handling
- [ ] Test

`max_abs_norm`
- [ ] Consistent handling
- [ ] Test

`minmax_norm`
- [ ] Consistent handling
- [ ] Test

`power_norm`
- [ ] Consistent handling
- [ ] Test

`quantile_norm`
- [ ] Consistent handling
- [ ] Test

`robust_scale_norm`
- [ ] Consistent handling
- [ ] Test

`scale_norm`
- [ ] Consistent handling
- [ ] Test

`offset_negative_values`
- [ ] Consistent handling
- [ ] Test

**Comment by Zethson** [comment]
@eroell this is done via https://github.com/theislab/ehrapy/pull/835 right? Closing now but please reopen if necessary.

**Comment by eroell** [comment]
Yes, its done via #835, I seem to have forgotten close it by linking
